node/pkg/watchers/evm/chain_config.go

```diff
@@ -0,0 +1,274 @@
+		vaa.ChainIDXLayer: {Finalized: true, Safe: true, EvmChainID: 196, PublicRPC: "https://xlayerrpc.okx.com"},
+
+		// As of 9/06/2024 Linea supports polling for finalized but not safe.
+		vaa.ChainIDLinea: {Finalized: true, Safe: false, EvmChainID: 59144, PublicRPC: "https://linea-rpc.publicnode.com"},
+
+		vaa.ChainIDBerachain: {Finalized: false, Safe: false, EvmChainID: 80094, PublicRPC: "https://berachain-rpc.publicnode.com"},
+		// vaa.ChainIDSeiEVM:     Not in Mainnet yet.
+		// vaa.ChainIDEclipse:    Not supported in the guardian.
+		// vaa.ChainIDBOB:        Not supported in the guardian.
+		vaa.ChainIDSnaxchain:  {Finalized: true, Safe: true, EvmChainID: 2192, PublicRPC: "https://mainnet.snaxchain.io"},
+		vaa.ChainIDUnichain:   {Finalized: true, Safe: true, EvmChainID: 130, PublicRPC: "https://unichain-rpc.publicnode.com"},
+		vaa.ChainIDWorldchain: {Finalized: true, Safe: true, EvmChainID: 480, PublicRPC: "https://worldchain-mainnet.g.alchemy.com/public"},
+		// vaa.ChainIDInk:        Not in Mainnet yet.
+		// vaa.ChainIDHyperEVM:   Not in Mainnet yet.
+		// vaa.ChainIDMonad:      Not in Mainnet yet.
+	}
+
+	// testnetChainConfig specifies the configuration for all chains enabled in Mainnet.
+	// NOTE: Only add a chain here if the watcher should allow it in Testnet.
+	// NOTE: If you change this data, be sure and run the tests described at the top of this file!
+	testnetChainConfig = EnvMap{
+		// For Ethereum testnet we actually use Holeksy since Goerli is deprecated.
+		vaa.ChainIDEthereum: {Finalized: true, Safe: true, EvmChainID: 17000, PublicRPC: "https://ethereum-holesky-rpc.publicnode.com"},
+		vaa.ChainIDBSC:      {Finalized: true, Safe: true, EvmChainID: 97, PublicRPC: "https://bsc-testnet-rpc.publicnode.com"},
+
+		// Polygon supports polling for finalized but not safe: https://forum.polygon.technology/t/optimizing-decentralized-apps-ux-with-milestones-a-significantly-accelerated-finality-solution/13154
+		vaa.ChainIDPolygon: {Finalized: true, Safe: false, EvmChainID: 80001}, // Polygon Mumbai is deprecated.
+
+		vaa.ChainIDAvalanche: {Finalized: false, Safe: false, EvmChainID: 43113, PublicRPC: "https://avalanche-fuji-c-chain-rpc.publicnode.com"},
+		vaa.ChainIDOasis:     {Finalized: false, Safe: false, EvmChainID: 42261, PublicRPC: "https://testnet.emerald.oasis.dev"},
+		// vaa.ChainIDAurora:    Not supported in the guardian.
+		vaa.ChainIDFantom:   {Finalized: false, Safe: false, EvmChainID: 4002, PublicRPC: "https://fantom-testnet-rpc.publicnode.com"},
+		vaa.ChainIDKarura:   {Finalized: true, Safe: true, EvmChainID: 596, PublicRPC: "https://eth-rpc-karura-testnet.aca-staging.network"},
+		vaa.ChainIDAcala:    {Finalized: true, Safe: true, EvmChainID: 597, PublicRPC: "https://eth-rpc-acala-testnet.aca-staging.network"},
+		vaa.ChainIDKlaytn:   {Finalized: false, Safe: false, EvmChainID: 1001}, // As of Feb 2025, can't find a working public node.
```

Comment:
KLAYTN_RPC="https://rpc.ankr.com/klaytn_testnet"

Reply:
This endpoint gives `ERROR: Failed to query EVM chain ID for test klaytn: failed to read evm chain id: API key is not allowed to access blockchain`.